src/main/java/gregtech/common/blocks/GT_TileEntity_Ores.java

```diff
@@ -41,7 +41,7 @@ public static boolean setOreBlock(World aWorld, int aX, int aY, int aZ, int aMet
         return setOreBlock(aWorld, aX, aY, aZ, aMetaData, isSmallOre, false);
     }
 
-    public static boolean setOreBlock(World aWorld, int aX, int aY, int aZ, int aMetaData, boolean isSmallOre, boolean air) { //TODO
+    public static boolean setOreBlock(World aWorld, int aX, int aY, int aZ, int aMetaData, boolean isSmallOre, boolean air) {
         if (!air) {
```

Comment:
Need to redo the air boolean logic.  It does not actually enable what you would think, ie placing ores in air. It also causes problems with placing ores above Y=128 in the Deep dark since getActualHeight does not correlate with the real max height of the dimension.  getActualHeight is based on hasNoSky(which is a terribly incorrect name, it should instead be called isNether or smallHeight).  My suggestion is to remove the air boolean completely. Validating ores place at legal heights should be done when reading the config file, or not at all (my preference).